packages/@aws-cdk/aws-synthetics/test/integ.canary.ts

```diff
@@ -60,6 +60,15 @@ new synthetics.Canary(stack, 'MyCanaryThree', {
   runtime: synthetics.Runtime.SYNTHETICS_NODEJS_PUPPETEER_3_3,
 });
 
+new synthetics.Canary(stack, 'MyCanaryFour', {
```

Comment:
if you change an integ test you are going to have to update the corresponding expected.json file by running `yarn integ -t integ.canary.js`. But i don't think we need that for such a small change. i prefer if you just remove the changes to the integ test.

Reply:
>  prefer if you just remove the changes to the integ test.

Removed in 90c89a4a1